### PR TITLE
Enable live reloading

### PR DESCRIPTION
**Pull request by Bruception**
### Description

- Uses Webpack dev server with live reloading

https://user-images.githubusercontent.com/58147810/159149022-bb224da5-87c7-4e33-adb2-4c1c22852ed4.mov

### CODE REVIEW

File: frontend/webpack/config.base.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ const BASE_CONFIGURATION = {
     extensions: [".ts", ".js"],
   },
   output: {
-    path: path.resolve(__dirname, "../public/js/"),
-    filename: "monkeytype.js",
+    filename: "./js/monkeytype.js",
```
Comment (Bruception) [inline frontend/webpack/config.base.js:19]: This is relative to the path defined below.

File: frontend/package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "dep-graph": "madge -c -i \"dep-graph.png\" ./src/scripts",
     "build": "npx gulp build",
     "build:live": "npx gulp build-production",
-    "start:dev": "concurrently \"npm run build\" \"firebase serve --only hosting\" \"npx gulp watch\"",
+    "start:dev": "concurrently \"webpack serve --config=./webpack/config.dev.js\" \"npx gulp watch\"",
```
Comment (Bruception) [inline frontend/package.json:10]: Thinking about just naming this script to `dev`

Comment (Miodec) [reply frontend/package.json:10]: Sure, why not. Be sure to update contributing.md and add an echo to start:dev that the command changed.

**Review by Miodec — CHANGES_REQUESTED** [review]
As mentioned in the comment - please update the script names and contributing.md

**Comment by Bruception** [comment]
> As mentioned in the comment - please update the script names and contributing.md

I don't think it makes sense to keep the `start:dev` script. People can just read the package file.

**Review by Miodec — APPROVED** [review]

**Pull request merged**